front/abbrevs.tex

```diff
@@ -1,9 +1,79 @@
-% Just write down your abbervations here - or comment-out the command
-
-\abbreviationsAndNotation{
-\begin{tabular}{p{2cm}@{:\quad}l}
-QED & quod erat demonstrandum (``what was to be shown'')\\
-$c$ & the speed of light \\
-$a\pm b$ & the closed interval $\left[a-b,a+b\right]$ \\
-\end{tabular}
+% Use this file to create "glossary entries" for abbreviations and acronyms,
+% and other notation. The entries defined here don't necessarily have to be 
+% used in the thesis (but then you have to decide whether or not to display
+% the unused entries).
+
+% For this file to compile (and the example text in the main/prelims.tex file),
+% the package glossaries-extra is required. It is automatically included unless
+% the noabbrevs class option is used.
+
+% The following will alter the style for typesetting abbreviations when using 
+% the \gls command. Note you can also use multiple styles by categorizing 
+% abbreviations; see the documentation for the glossaries-extras package at:
+% https://ctan.org/pkg/glossaries-extra
+%
+%\setabbreviationstyle[acronym]{long-short-sc}
+%
+% If you're wondering why we're setting the seemingly-redundant "notation 
+% category", that's a hack discussed here:
+% https://tex.stackexchange.com/q/630541/5640 
```

Comment:
Perhaps it is also good to mention here that without the category, all pages where the term appear would be indexed?